scripts/ruby/lib/parser_options/es_alias_add.rb

```diff
@@ -0,0 +1,43 @@
+module Parser
+  def self.es_alias_add
+    @usage = "Usage: ruby scripts/ruby/es_alias_add.rb -a alias -i index"
+    options = {}
+
+    optparse = OptionParser.new do |opts|
+      opts.banner = @usage
+
+      opts.on( '-h', '--help', 'How does this work?') do
+        puts opts
+        exit
+      end
+
+      options["alias"] = nil
+      opts.on( '-a', '--alias [input]', 'Alias (cdrhapi-v1') do |input|
+        if input && input.length > 0
+          options["alias"] = input
+        else
+          puts "Must specify an alias with -a flag"
+          exit
+        end
+      end
+
+      options["index"] = nil
+      opts.on( '-i', '--index [input]', 'Index (cdrhapi-v1.1') do |input|
+        if input && input.length > 0
+          options["index"] = input
+        else
+          puts "Must specify an index with -i flag"
+          exit
+        end
+      end
+
+    end
+
+    optparse.parse!
+    if options["alias"].nil? || options["index"].nil?
```

Comment:
Do we need to watch for empty strings too? I don't remember if Ruby assigns it like that if one puts the option flag but no value after it. Parser probably prevents that, right? Just want to be extra careful about creating URL strings without a part and wiping out all names or something 😬

Reply:
These should be getting caught in the above steps, actually.  So if somebody enters `-a` and nothing after it, a couple lines above the parser will have caught it and exited the program.  I'm just checking here if they were ever set at all, so hopefully `nil?` is enough of a check?